src/com/twu/biblioteca/MainMenu.java

```diff
@@ -45,26 +46,26 @@ public void showNoBooksMessage() {
 
 
     public void showOptions() {
-        for (int i = 0; i < options.size(); i++) {
-            System.out.println(options.get(i));
+        System.out.print(selectOption + "\n");
+        for (String option : options) {
+            System.out.println(option);
         }
         System.out.print(">");
     }
 
     public void manageOptions() {
-        int counter = 0;
         Scanner optionsScanner = new Scanner(System.in);
-
-        while (counter == 0) {
+            while (true) {
```

Comment:
Yes... and no.  Using infinite loops and having to explicitly break out of them (in this case, using `System.exit`) is generally considered a bit lazy.  I like using a boolean to control the loop, but could it be a variable which is changed when the user selects `3` instead?